app/src/main/java/com/swent/assos/ui/components/NewsItem.kt

```diff
@@ -40,7 +40,7 @@ fun NewsItem(news: News, navigationActions: NavigationActions) {
               .border(width = 0.5.dp, color = Color.LightGray, shape = RoundedCornerShape(12.dp))) {
         Column(
             modifier =
-                Modifier.width(200.dp).padding(vertical = 0.dp).clickable {
+                Modifier.width(200.dp).padding(vertical = 0.dp).clickable(false) {
```

Comment:
Leave a ToDo comment so that we remember to change the fact that the component is not clickable